### DISCUSSION TITLE
Preline JavaScript bundle not working

### DISCUSSION THREAD

**Discussion by johannesteshome** [htmlstreamofficial/preline#20] [General]
Hey everyone, I tried to add a dropdown component in my vue project(tailwindCSS installed in NPM) but its functionality is not working. I followed the documentation's instruction but it doesn't seem to help. Anyone can you help me with this?

**Comment by bernardyamoah** [comment]
I also encountered the same issue

**Comment by ankurdhamija83** [comment]
I am also facing the same issue. The js bundle is not working.
Can someone please help?

Regards,

**Comment by jahaganiev** [comment]
Hey @John-droid-dotcom, @bernardyamoah - you may check out discussion #70 for other comments and workarounds.

**Reply by bernardyamoah** [reply]
Sure, thanks!
